servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcStatus.java

```diff
@@ -70,25 +78,38 @@ public GrpcStatus(GrpcStatusCode code, @Nullable Throwable cause) {
      * @param code status code.
      * @param cause cause.
      * @param description additional description.
+     * @deprecated Use {@link GrpcStatusException} if there is a cause or
+     * {@link GrpcStatus#GrpcStatus(GrpcStatusCode, CharSequence)}.
      */
+    @Deprecated
     public GrpcStatus(GrpcStatusCode code, @Nullable Throwable cause, @Nullable CharSequence description) {
         this.code = requireNonNull(code);
         this.cause = cause;
         this.description = description == null ? null : description.toString();
     }
 
+    /**
+     * Constructs a status with cause and additional description.
+     * @param code status code.
+     * @param description additional description.
+     */
+    public GrpcStatus(GrpcStatusCode code, @Nullable CharSequence description) {
+        this.code = requireNonNull(code);
+        this.cause = null;
+        this.description = description == null ? null : description.toString();
```

Comment:
If it's always a `String` internally, consider either changing the input argument to `String` or adding an alternative `description()` method that returns `CharSequence`.